apache/rocketmq/v1/definition.proto

```diff
@@ -0,0 +1,311 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/duration.proto";
+
+package apache.rocketmq.v1;
+
+option java_multiple_files = true;
+option java_package = "apache.rocketmq.v1";
+option java_generate_equals_and_hash = true;
+option java_string_check_utf8 = true;
+
+// Ali Cloud Service
+option java_outer_classname = "MQDomain";
+
+enum Permission {
+  NONE = 0;
+  READ = 1;
+  WRITE = 2;
+  READ_WRITE = 3;
+
+  reserved 4 to 64;
+}
+
+enum FilterType {
+  TAG = 0;
+  SQL = 1;
+
+  reserved 2 to 64;
+}
+
+message FilterExpression {
+  FilterType type = 1;
+  string expression = 2;
+
+  reserved 3 to 64;
+}
+
+// Dead lettering is done on a best effort basis. The same message might be
+// dead lettered multiple times.
+//
+// If validation on any of the fields fails at subscription creation/update,
+// the create/update subscription request will fail.
+message DeadLetterPolicy {
+  // The maximum number of delivery attempts for any message.
+  //
+  // This field will be honored on a best effort basis.
+  //
+  // If this parameter is 0, a default value of 5 is used.
```

Comment:
The default value here is incorrect. Default max attempt is 16

Reply:
Good catch, resolved~